bitcoin/Cargo.toml

```diff
@@ -36,10 +36,10 @@ features = [ "std", "secp-recovery", "base64", "rand", "serde", "bitcoinconsensu
 rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
-bitcoin-internals = { path = "../internals" }
+bitcoin-internals = { git = "https://github.com/tcharding/rust-bitcoin", branch = "01-20-rename-hashes-methods" }
 bech32 = { version = "0.9.0", default-features = false }
 bitcoin_hashes = { version = "0.11.0", default-features = false }
-secp256k1 = { version = "0.25.0", default-features = false, features = ["bitcoin_hashes"] }
+secp256k1 = { git = "https://github.com/tcharding/rust-secp256k1", branch = "01-31-rename-hashes-methods", default-features = false, features = ["bitcoin_hashes"] }
```

Comment:
This looks fine, though I'd use a tag rather than a branch because you might want to clean up your list of branches one day, but tags probably you'll never delete.

Reply:
Sweet, tag it is. Can you post it please once you've done it so I can use it here (I can't push to `rust-bitcoin/rust-secp256k1`)

Reply:
Pushed. Tag is `2023-02-01--fix-hashes`.